### ISSUE TITLE
androidenv: duplicate entries in generated file addons.nix

### ISSUE THREAD

**Issue by k0001** [NixOS/nixpkgs#95512]
**Describe the bug**

The `pkgs/development/mobile/androidenv/generated/addons.nix` file, generated by `pkgs/development/mobile/androidenv/generate.sh`, includes duplicate entries. 

These duplicates cause the following error: `attribute 'name' at /xxx/nixpkgs.git/pkgs/development/mobile/androidenv/generated/addons.nix:622:7 already defined at /xxx/nixpkgs.git/pkgs/development/mobile/androidenv/generated/addons.nix:638:7`

Here's an example duplicate entry: 
https://github.com/NixOS/nixpkgs/blob/99d379c45c793c078af4bb5d6c85459f72b1f30b/pkgs/development/mobile/androidenv/generated/addons.nix#L659-L688

**To Reproduce**

```bash
nix repl ./pkgs/development/mobile/androidenv/generated/addons.nix
Welcome to Nix version 2.3.6. Type :? for help.

Loading './pkgs/development/mobile/androidenv/generated/addons.nix'...
error: attribute 'name' at /xxx/nixpkgs.git/pkgs/development/mobile/androidenv/generated/addons.nix:622:7 already defined at /xxx/nixpkgs.git/pkgs/development/mobile/androidenv/generated/addons.nix:638:7
```

**Expected behavior**

There shouldn't be duplicate entries in `addons.nix`. That is, `generate.sh` should not generate duplicate entries.

**Notify maintainers**
@svanderburg @tadfisher 

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: androidenv
# a list of nixos modules affected by the problem
module:
```

**Comment by matthewbauer** [comment]
I think the script to generate it is broken. I have a manual fix here:

https://github.com/NixOS/nixpkgs/pull/95309/commits/4cbca0893af3b4bdac508894fb6d9b10040a5179

**Comment by numinit** [comment]
Should be solved by #89775 now.
